### DISCUSSION TITLE
How to save a new rst updated via pydpf-core?

### DISCUSSION THREAD

**Discussion by fceyhuns** [ansys/pydpf-core#989] [Q&A]
I have this sample script: 

```
import ansys.dpf.core as dpf
import numpy as np
rstFile = r'file.rst'
result = dpf.Model(rstFile)
results = result.results
dispN = results.displacement
dc = dispN.on_all_time_freqs.eval()
data6 = []
for x in range(len(dc[0].data)):
    data6.append(np.random.random(3))
    
data6 = np.array(data6)
dc[0].data = data6
```

When I animate dc after this update, I can see the random values assigned to the first time frame. However, I cannot save these updated results as a new *.rst file. Is there any way to save a new rst file in the API?

**Comment by PProfizi** [comment]
Hello @fceyhuns,
Sadly DPF does not have the capability to modify Ansys result files at the moment.
What we provide for now is a way of extracting results and post-processing them.
We do however enable export to different data formats using some of our [serialization operators](https://dpf.docs.pyansys.com/version/stable/api/ansys.dpf.core.operators.serialization.html).

**Comment by fceyhuns** [comment]
Thank you for you response. I am also using ansys-mapdl-reader. It is limited compared to dpf, am I right? 

Can I write only selected (only the ones I update) outputs to vtk using serialization operators?

**Comment by PProfizi** [comment]
Hello @fceyhuns,
As far as I am aware, the `ansys-mapdl-reader` is actually deprecated and not supported anymore for newer Ansys versions.

Using the [vtk_export](https://dpf.docs.pyansys.com/version/0.5/api/ansys.dpf.core.operators.serialization.vtk_export.html) operator, you should be able to filter the fields you export to vtk by setting the fields1, fields2, ...etc pins.
Either construct a FieldsContainer with only the fields of interest and connect it to `fields1` pin, or select a given field by connecting it directly as `fields1`.

**Comment by fceyhuns** [comment]
Thank you for your response
